paket.dependencies

```diff
@@ -43,7 +43,7 @@ nuget Be.Vlaanderen.Basisregisters.ProjectionHandling.Runner 9.0.0
 nuget Be.Vlaanderen.Basisregisters.ProjectionHandling.Connector 9.0.0
 nuget Be.Vlaanderen.Basisregisters.ProjectionHandling.Connector.Testing 9.0.0
 
-nuget Be.Vlaanderen.Basisregisters.Projector 10.1.1
+nuget Be.Vlaanderen.Basisregisters.Projector 11.0.0
```

Comment:
do we still need this dependency? why?

Reply:
I've used the projector here so we can communicate with it. To get the status, ask to start/stop, etc...